src/components/fpl/ControlsBar.tsx

```
@@ -0,0 +1,153 @@
+            onKeyPress={handleKeyPress}
+            className="w-32 px-2 py-1 text-sm border border-gray-300 dark:border-gray-600 rounded focus:ring-2 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:text-white"
+            placeholder="133790"
+            title="Press Enter to load team"
+          />
+        </div>
+
+        {/* Gameweek Input */}
+        <div className="flex items-center gap-2">
+          <label className="text-sm font-medium text-gray-700 dark:text-gray-300 whitespace-nowrap">
+            GW:
+          </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)